### PR TITLE
support setting journalmatch

### PR DESCRIPTION
**Pull request by cwells**
#### Pull Request (PR) description

Add support for setting journalmatch.

#### This Pull Request (PR) fixes the following issues

In order to use systemd backend, you must be able to specify journalmatch.

### CODE REVIEW

File: manifests/jail.pp [hunk 1]
```diff
@@ -14,6 +14,7 @@
   Integer $bantime = $fail2ban::bantime,
   Optional[String] $port = undef,
   Optional[String] $backend = undef,
+  Optional[String] $journalmatch = undef,
```
[the remark this answers is not in the document]
Comment (cwells) [reply manifests/jail.pp]: journalmatch can be an arbitrary string, e.g. 

`journalmatch: _SYSTEMD_UNIT=haproxy.service`

[the remark this answers is not in the document]
Comment (bastelfreak) [reply manifests/jail.pp]: okay, so we should probably stick with my suggestion.

[the remark this answers is not in the document]
Comment (cwells) [reply manifests/jail.pp]: Actually, hold off. It appears I put this setting in the wrong location. It ought to be in filters.

**Comment by cwells** [comment]
Setting put in wrong config file.

**Pull request closed without merging**